$OpenBSD: patch-testing_acceptance_test_py,v 1.1.1.1 2016/09/01 12:56:17 edd Exp $

This test is fine, just it doesn't wait long enough.

--- testing/acceptance_test.py.orig	Wed Aug 31 16:03:03 2016
+++ testing/acceptance_test.py	Wed Aug 31 16:12:47 2016
@@ -240,12 +240,12 @@ class TestDistribution:
         testdir.makepyfile("""
             def test_sleep():
                 import time
-                time.sleep(10)
+                time.sleep(30)
         """)
         child = testdir.spawn_pytest("-n1 -v")
         child.expect(".*test_sleep.*")
         child.kill(2)  # keyboard interrupt
-        child.expect(".*KeyboardInterrupt.*")
+        child.expect(".*KeyboardInterrupt.*", timeout=28)
         # child.expect(".*seconds.*")
         child.close()
         # assert ret == 2
